pages/linux/poweroff.md

```diff
@@ -1,7 +1,12 @@
 # poweroff
 
 > Shutdown the system.
+> More information: <https://man7.org/linux/man-pages/man8/reboot.8.html>.
 
-- Poweroff the system:
+- Power off the system:
+
+`poweroff`
+
+- Send an ACPI hardware signal to initiate a complete and immediate shutdown:
```

Comment:
This new example doesn't add something new, only explain how poweroff works. I think you can work over the `--halt` vs `--poweroff` arguments to add something new, or explain `--wtmp-only`. etc.

Reply:
But isn't there a difference between `poweroff` and `sudo poweroff`?

Reply:
I think the difference is between `halt`, `poweroff` and `shutdown` shutdown sequences.